timer.js

```diff
@@ -0,0 +1,13 @@
+let now = 30;
```

Comment:
I would define this within the `timer()` function, otherwise if you had multiple timers they would interfere.